client/public/frame.html

```diff
@@ -47,5 +47,6 @@
 </head>
 <body>
   <main id="main"></main>
+  <div id="root"></div>
```

Comment:
I'm not totally following why this is required. Is the cookbook unable to use `main id="main"`? If not, and it seems important that we specifically use `<div id="root">`, then can we adjust the existing code (ie. the TryPureScript rendering) to point at this div and remove `<main id="main">`? It's a bit odd to have both.

Reply:
`main` is used to redirect `Console.log` (when under a `TryPureScript.render =<< TryPureScript.withConsole`, so that the strings logged to the console are instead added as nodes under the `main id="main"` HTML element rather than logging to the web browser's console.

I don't want the two to mix, so I'm creating a separate element for React usage.